src/lib.rs

```diff
@@ -65,6 +81,10 @@ module! {
         pub struct GetLogs as "eth_getLogs"
             (LogFilter,) => Vec<Log>;
 
+        /// Returns the value from a storage position at a given address.
+        pub struct GetStorageAt as "eth_getStorageAt"
+            (Address, U256, Option<BlockSpec>) => [u8; 32] [serialization::bytearray];
```

Comment:
I checked the spec and it looks like a block hash is also accepted here, so it should be:

```suggestion
            (Address, U256, Option<BlockId>) => [u8; 32] [serialization::bytearray];
```

<img width="636" alt="image" src="https://github.com/nlordell/ethrpc-rs/assets/4210206/2d59f08c-dca4-42fc-81b0-a4b7433206a7">


Can you please look over the other methods and make sure this isn’t the only one you got wrong? I assume methods like `eth_getBalance` would also accept a block hash not not just a tag or number.

Reply:
When I look at them, I was comparing with the "Block" structure they accept in comparison with what we passed to getblockReceipts. I can look again, but maybe that one was wrong or maybe the docs use "Block" for different purposes.

Reply:
> but maybe that one was wrong or maybe the docs use "Block" for different purposes.

They definitely do:

<img width="1123" alt="image" src="https://github.com/nlordell/ethrpc-rs/assets/4210206/fb57711c-73fc-4a3a-8c53-0382af0c49d8">

Another different use of it:

<img width="1164" alt="image" src="https://github.com/nlordell/ethrpc-rs/assets/4210206/506cfb57-2206-403a-8182-cc4ded49c556">

Reply:
So it looks like “Block” can either mean `number + tag` or `number + tag + hash`.

Reply:
Okie. I will pass through the items here.